.gitignore

```diff
@@ -8,6 +8,9 @@
 # "git add -f". See "git help gitignore" for more information.
 
 
+# Virtualenv
+/.venv/
```

Comment:
Well... this is your configuration. Maybe you should add `/.venv/.gitignore` with `*` in it, instead of editing here.

Reply:
@Upabjojr , Its alright, nobody can programming without bug.

@Upabjojr , would you like to add this configuration to the public gitignore.

Because, there are a number of people who prefer virtual environment this way. GitHub also put this in its recommend gitignore list [Python.gitignore](https://github.com/github/gitignore/blob/master/Python.gitignore).

And it doesn't affect people who don't use virualenv this way.